file-formats/tabl/type/zif_aff_indx_v1.intf.abap

```diff
@@ -0,0 +1,42 @@
+INTERFACE zif_aff_indx_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Index Fields</p>
+    "! Index fields
+    BEGIN OF ty_indexfield,
+      "! <p class="shorttext">Field Name</p>
+      "! Field name
+      fieldname TYPE c LENGTH 30,
```

Comment:
I guess the field name should be required, shouldn't it?

Minor: I also suggest to rename it to field_name so that we get the key "fieldName"

```suggestion
      "! $required
      field_name TYPE c LENGTH 30,
```